browser-sign-in/OktaBrowserSignIn/WelcomeViewController.swift

```diff
@@ -88,16 +106,16 @@ private extension WelcomeViewController {
     var configForUITests: [String: String]? {
```

Comment:
This code can move into the AppDelegate if you centralize configuration of the WebAuthentication instance.